SwiftDashboard.md

```diff
@@ -0,0 +1,57 @@
+# Firebase Swift Modernization Dashboard
+
+This dashboard summarizes the status of Firebase's [2022 Swift Modernization Project](Roadmap.md).
+Please upvote or create a [feature request](https://github.com/firebase/firebase-ios-sdk/issues)
+to help prioritize any particular cell(s).
+
+|                       | AB  | An     | ApC    | ApD    | Aut    | Cor    | Crs    | DB     | Fst    | Fn     | IAM    | Ins    | Msg    | MLM    | Prf    | RC     |    Str |
+|   :---                | :--- | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: |
+| **Swift Library**     | ❌   |   ✔   | ❌     |❌     | ❌     | ❌     | ❌      |  ✔     |  ✔    | 1      |  ✔     | ❌    | ❌     | ✔      | ❌     | ❌    | ✔     |
+| **API Tests**         | ❌   |  ❌    |  ✔    |❌     | ❌     | ✔       | ❌     | 3      | 2     |  ✔     | 2      | ❌     | ❌     | 2      | ❌    |  ✔     | ✔    |
```

Comment:
Installations can have a check for API Tests: Installations API build tests (#8353)

Reply:
Good catch. Thanks! Fixed.